src/test/java/spec/yandexSpec/CreateAccount.java

```diff
@@ -18,46 +18,49 @@
 
 public class CreateAccount extends BaseSpec {
```

Comment:
I don't like that all tests are verifying... titles/URL. Can you make tests more real?

Reply:
I added different types of asserts.